debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/connection/MongoDbConnection.java

```diff
@@ -175,6 +174,19 @@ public List<CollectionId> collections() throws InterruptedException {
         });
     }
 
+    /**
+     * Executes the ping command (<a href="https://www.mongodb.com/docs/manual/reference/command/ping/">...</a>) using
+     * the first available database
+     *
+     * @return timestamp of the executed operation
+     */
+    public BsonTimestamp ping() throws InterruptedException {
+        return execute("ping on first available database", client -> {
+            var dbName = databaseNames().stream().findFirst().orElse("admin");
```

Comment:
This is required as system databases are filtered out of `databaseNames()` call so we might get an enpty collection. 

This won't happen when `capture.scope` is set to `database` as in such case the method return the value specified in `capture.target` propertery. However executing the ping command is possible even when the user doesn't have the read permissions for the "admin" database.